conf/fluent-bit.conf

```diff
@@ -45,4 +45,4 @@
 
 [OUTPUT]
     Name  stdout
-    Match *
+    Match * 
```

Comment:
revert this change

Reply:
@edsiper It is the same as master, have copied it multiple times I don't know why it is showing diff, I have removed the space also.